### DISCUSSION TITLE
FYI: there is empty package at https://pypi.org/project/pinecone-io/

### DISCUSSION THREAD

**Discussion by niki-sp** [reflex-dev/reflex#218] [General]
This can lead to attacks

**Comment by Alek99** [comment]
Hi, Im not sure if I am following our package is `pynecone-io`, **not** `pinecone-io`

**Reply by Alek99** [reply]
The other package is not related to us so we have no control over it

**Reply by FHU-yezi** [reply]
He means if someone upload bad code to this fake package, and user has a typo in their command, hacker can run any code they want.
